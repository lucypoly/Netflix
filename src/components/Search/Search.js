import React from 'react';
import PropTypes from 'prop-types';

const Search = ({ title }) => (
  <h1 className="pink search">{title}</h1>
);

Search.propTypes = {
  title: PropTypes.string.isRequired,
};

export default Search;
