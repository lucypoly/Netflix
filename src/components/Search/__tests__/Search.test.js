import React from 'react';
import renderer from 'react-test-renderer';
import Search from '../Search';

let props;

describe('Search', () => {
  beforeEach(() => {
    props = {
      title: 'Search title',
    };
  });

  test('renders correctly', () => {
    const instance = renderer.create(<Search {...props} />);

    expect(instance.toJSON()).toMatchSnapshot();
  });
});
