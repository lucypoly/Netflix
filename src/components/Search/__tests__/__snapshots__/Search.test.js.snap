// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<h1
  className="pink"
>
  Search title
</h1>
`;
